provisiondomainmanager_policy.tf

```diff
@@ -0,0 +1,527 @@
+      "ecr:ListImages",
+      "ecr:PutImage",
+      "ecr:PutLifecyclePolicy",
+      "ecr:SetRepositoryPolicy",
+      "ecr:UploadLayerPart"
+    ]
+
+    resources = [
+      "arn:aws:ecr::${local.userservices_account_id}:repository/domain-manager-*"
+    ]
+  }
+
+  statement {
+    actions = [
+      "ecr:DescribeRegistry",
+      "ecr:DescribeRepositories",
+      "ecr:GetAuthorizationToken",
+      "ecr:ListTagsForResource",
+      "ecr:TagResource",
+      "ecr:UntagResource"
+    ]
+
+    resources = ["*"]
+  }
+
+  statement {
+    actions = [
+      "ecs:CreateService",
+      "ecs:DeleteCluster",
+      "ecs:DeleteService",
+      "ecs:DeregisterContainerInstance",
+      "ecs:ListContainerInstances",
+      "ecs:ListServices",
+      "ecs:ListTaskDefinitionFamilies",
+      "ecs:ListTaskDefinitions",
+      "ecs:ListTasks",
+      "ecs:RegisterContainerInstance",
+      "ecs:RunTask",
+      "ecs:StartTask",
+      "ecs:StopTask",
+      "ecs:SubmitContainerStateChange",
+      "ecs:UpdateContainerAgent",
+      "ecs:UpdateContainerInstancesState",
+      "ecs:UpdateService",
+      "ecs:UpdateServicePrimaryTaskSet",
+      "elasticloadbalancing:AddListenerCertificates",
+      "elasticloadbalancing:AddTags",
+      "elasticloadbalancing:AddTags",
+      "elasticloadbalancing:CreateListener",
+      "elasticloadbalancing:CreateLoadBalancer",
+      "elasticloadbalancing:CreateRule",
+      "elasticloadbalancing:CreateTargetGroup",
+      "elasticloadbalancing:DeleteListener",
+      "elasticloadbalancing:DeleteLoadBalancer",
+      "elasticloadbalancing:DeleteRule",
+      "elasticloadbalancing:DeleteTargetGroup",
+      "elasticloadbalancing:DeregisterTargets",
+      "elasticloadbalancing:ModifyListener",
+      "elasticloadbalancing:ModifyLoadBalancerAttributes",
+      "elasticloadbalancing:ModifyRule",
+      "elasticloadbalancing:ModifyTargetGroup",
+      "elasticloadbalancing:ModifyTargetGroupAttributes",
+      "elasticloadbalancing:RegisterTargets",
+      "elasticloadbalancing:RemoveListenerCertificates",
+      "elasticloadbalancing:RemoveTags",
+      "elasticloadbalancing:RemoveTags",
+      "elasticloadbalancing:SetIpAddressType",
+      "elasticloadbalancing:SetRulePriorities",
+      "elasticloadbalancing:SetSecurityGroups",
+      "elasticloadbalancing:SetSubnets",
+      "elasticloadbalancing:SetWebAcl",
+      "events:DeleteRule",
+      "events:DescribeRule",
+      "events:DisableRule",
+      "events:EnableRule",
+      "events:ListRuleNamesByTarget",
+      "events:ListRules",
+      "events:ListTagsForResource",
+      "events:ListTargetsByRule",
+      "events:PutEvents",
+      "events:PutPermission",
+      "events:PutRule",
+      "events:PutTargets",
+      "events:RemovePermission",
+      "events:RemoveTargets",
+      "events:TagResource",
+      "events:UntagResource"
+    ]
+
+    resources = [
+      "arn:aws:ecs::${local.userservices_account_id}:cluster/domain-manager-*",
```

Comment:
If it were me, I would break things like this up into three statement blocks - one for each of `ecs`, `elasticloadbalancing`, and `events`.

Reply:
Excellent suggestion- implemented in https://github.com/cisagov/cool-accounts-userservices/pull/3/commits/5ccc0ec2729f22aa263a9077ca86236e1959e0c3.  Thanks!